src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -100,6 +100,20 @@ public override void ExecuteCmdlet()
                                 flexibleOrchestrationModeDefaultParameters(parameters);
                                 checkFlexibleOrchestrationModeParamsDefaultParamSet(parameters);
                             }
+                            
+                            if (parameters.VirtualMachineProfile?.SecurityProfile?.SecurityType == "TrustedLaunch" || parameters.VirtualMachineProfile?.SecurityProfile?.SecurityType =="ConfidentialVM")
+                            {
+                                if (parameters.VirtualMachineProfile?.SecurityProfile?.UefiSettings != null)
+                                {
+                                    parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.SecureBootEnabled = parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.SecureBootEnabled != null ? parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.SecureBootEnabled : true;
+                                    parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.VTpmEnabled = parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.VTpmEnabled != null ? parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.VTpmEnabled : true;
```

Comment:
This can be shortened to `parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.VTpmEnabled = parameters.VirtualMachineProfile.SecurityProfile.UefiSettings.VTpmEnabled ?? true;`